src/tap/src/tapcell.tcl

```diff
@@ -250,890 +222,15 @@ proc tapcell_ripup { args } {
   set endcaps_removed [tap::remove_cells $endcap_prefix]
   utl::info TAP 101 "Endcaps removed: $endcaps_removed"
 
-  # Reset global parameters
-  set tap::phy_idx 0
-  set tap::filled_sites []
+  # Reset global parameters 
+  tap::reset
 }
 
 namespace eval tap {
-variable phy_idx
-variable filled_sites
 variable default_tapcell_prefix "TAP_"
 variable default_endcap_prefix "PHY_"
```

Comment:
I think you can remove the whole `namespace eval tap` section

Reply:
If I do so is it ok to pass directly the values of the default_tapcell_prefix and default_endcap_prefix in the tcl code?